src/Sign.SignatureProviders.KeyVault/Resources.resx

```diff
@@ -117,6 +117,9 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="CertificateDetails" xml:space="preserve">
+    <value>Certificate Details:</value>
```

Comment:
```suggestion
    <value>Certificate details:</value>
```

This is to be consistent with https://github.com/dotnet/sign/pull/808/files#diff-04854b03e08087d8a503b4b9480e748e19826ee8603241c1cdb3eeb05eaaf80dR121.

Be sure to regenerate .xlf files.